streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -122,7 +123,8 @@ public Builder(final String storeName, final Serde<K> keySerde, final Serde<V> v
 
         @Override
         public StoreBuilder<InMemoryTimeOrderedKeyValueBuffer<K, V>> withLoggingEnabled(final Map<String, String> config) {
-            throw new UnsupportedOperationException();
```

Comment:
We should at least extend `InMemoryTimeOrderedKeyValueBufferTest`.

Wondering if we need other tests? \cc @vvcephei